apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -124,23 +124,23 @@ public DataSource dataSource(
       @Value("${" + PROP_DB_CONNECTIONS_MAX + ":-1}") String connectionsMaxText,
       @Value("${" + PROP_DB_SCHEMA_APPLY + ":false}") String schemaApplyText,
       MetricRegistry metricRegistry) {
-    HikariDataSource poolingDataSource;
+    DataSource poolingDataSource;
```

Comment:
The changes here are primarily to allow a non-HSQL datasource to return from createTestDatabase; since psql uses a PGSimpleDataSource instead of a HikariDataSource.